app/lib/features/events/pages/event_details_page.dart

```diff
@@ -635,40 +666,20 @@ class _EventDetailPageConsumerState extends ConsumerState<EventDetailPage> {
     );
   }
 
-  void showEditDescriptionSheet(String? formattedText, String? bodyText) {
+  void showEditDescriptionSheet(CalendarEvent ev) {
+    TextMessageContent? content = ev.description();
     showEditHtmlDescriptionBottomSheet(
       context: context,
-      descriptionHtmlValue: formattedText,
-      descriptionMarkdownValue: bodyText,
+      descriptionHtmlValue: content?.formatted(),
```

Comment:
`content?.formatted()`
This code is seems like not working correctly, It is causing issue on our HtmlEdtior.